src/main/java/com/example/onlinestore/user/User.java

```diff
@@ -0,0 +1,66 @@
+package com.example.onlinestore.user;
```

Comment:
[nitpick] domains usually use plural form:
```suggestion
package com.example.onlinestore.users;
```

Reply:
got it.